src/operations.jl

```diff
@@ -52,7 +52,10 @@ function amr_get(amr::JSON3.Object, ::Type{ODESystem})
         push!(eqs, ofunc ~ expr)
     end
 
-    structural_simplify(ODESystem(eqs, t, allfuncs, paramvars; defaults = [statefuncs .=> initial_vals; sym_defs], name=Symbol(amr.name)))
+    sys = structural_simplify(ODESystem(eqs, t, allfuncs, paramvars; defaults = [statefuncs .=> initial_vals; sym_defs], name=Symbol(amr.name)))
+    @info sys
```

Comment:
I've been trying to include at least the function name that the `@info` is contained in for ease of lookup.

Reply:
```suggestion
    @info "amr_get(amr, ODESystem) --> $sys"
```